packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -728,6 +730,20 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 				),
 			);
 		}
+		if (this.redirectTable.size > 0) {
+			const redirectTree = new SummaryTreeBuilder(blobsTreeName);
```

Comment:
this is the core of the change, just trying to move the redirect table from an opaque blob (above) to a summary tree. this keeps the blobs as attachment blobs.  A change like this required changes to all layers, so very costly to rollout